addons/web/static/src/views/fields/priority/priority_field.js

```diff
@@ -69,14 +69,16 @@ PriorityField.template = "web.PriorityField";
 PriorityField.props = {
     ...standardFieldProps,
     tooltipLabel: { type: String, optional: true },
+    autosave: { type: Boolean, optional: true },
 };
 
 PriorityField.displayName = _lt("Priority");
 PriorityField.supportedTypes = ["selection"];
 
-PriorityField.extractProps = ({ field }) => {
+PriorityField.extractProps = ({ field, attrs }) => {
     return {
         tooltipLabel: field.string,
+        autosave: "auto_save" in attrs.options ? !!attrs.options.auto_save : true,
```

Comment:
in https://github.com/odoo/odoo/pull/117103, the option is named `autosaved`. Could you adapt here for the sake of consistency? Thanks